jerry-debugger/jerry-client-ws.py

```diff
@@ -826,14 +835,30 @@ def parse_source(debugger, data):
     # Try to set the pending breakpoints
     if len(debugger.pending_breakpoint_list) != 0:
         logging.debug("Pending breakpoints list: %s", debugger.pending_breakpoint_list)
+        indexes = []
+        bp_list = debugger.pending_breakpoint_list
+
+        for breakpoint in bp_list:
+            for src in debugger.function_list.values():
+                if src.source_name == bp_list[breakpoint].source_name:
```

Comment:
Does function breakpoints has source names? Is it an empty string?

Reply:
Function breakpoints don't have source names.